pyccel/parser/semantic.py

```diff
@@ -3498,16 +3491,24 @@ def _visit_ClassDef(self, expr):
         #      - wouldn't be better if it is done inside ClassDef?
 
         name = expr.name
+        # remove quotes for str representation
         name = name.replace("'", '')
+
+        parent = [self.scope.find(s, 'classes') for s in expr.superclass]
+        if any(s is None for s in parent):
+            for s in parent:
+                if s is None:
+                    errors.report("Couldn't find class {s} in scope", symbol=expr,
+                            severity='error')
```

Comment:
Are you not missing an `f` in front of the string?

Actually, even if you add it this will print `Couldn't find class None in scope`, which is not very useful.

Reply:
Fixed 5cb4c09b